scripts/benchmark/trainer-benchmark.py

```diff
@@ -186,11 +186,11 @@ def get_base_command(args, output_dir):
     args.base_cmd = re.sub(r"[\\\n]+", " ", args.base_cmd)
 
     # remove --output_dir if any and set our own
-    args.base_cmd = re.sub("--output_dir\s+[^\s]+", "", args.base_cmd)
+    args.base_cmd = re.sub(r"--output_dir\s+[^\s]+", "", args.base_cmd)
```

Comment:
Same here - why these changes in this file?

Reply:
I don't know why these codes are changed. but when I run the ruff command that time it's automatically added.

Reply:
Now I solved it.